go/chat/sender.go

```diff
@@ -41,7 +41,7 @@ func NewBlockingSender(g *globals.Context, boxer *Boxer, getRi func() chat1.Remo
 		DebugLabeler:      utils.NewDebugLabeler(g.GetLog(), "BlockingSender", false),
 		getRi:             getRi,
 		boxer:             boxer,
-		store:             attachments.NewS3Store(g.GetLog(), g.GetEnv(), g.GetRuntimeDir()),
+		store:             attachments.NewS3Store(g.GlobalContext, g.GetLog(), g.GetEnv(), g.GetRuntimeDir()),
```

Comment:
Passing `g.GlobalContext` might undo some separation that @patrickxb had in mind, so there might be a slightly better way, not sure though.

Reply:
It's fine if it's necessary.